src/app/webserver/app-pools/identity.component.ts

```diff
@@ -27,7 +27,10 @@ import { ApplicationPoolIdentity } from './app-pool';
                 </fieldset>
                 <fieldset *ngIf="!!(_password)" class='inline-block'>
                     <label>Confirm Password</label>
-                    <input class="form-control" type="password" ngModel (ngModelChange)="onConfirmPassword($event)" [validateEqual]="_password" />
+                    <input class="form-control" type="password" [(ngModel)]="_confirm" (ngModelChange)="onConfirmPassword($event)" [validateEqual]="_password" />
+                </fieldset>
+                <fieldset *ngIf="!!(_confirm) && _confirm !== _password" class='inline-block error-message'>
```

Comment:
try to do this without fieldset

Reply:
@shhsu Thanks for the feedback. The fieldset is now removed.